prime-router/src/main/java/fhirengine/translation/hl7/structures/nistelr251/datatype/CE.java

```diff
@@ -0,0 +1,208 @@
+/*
+ * This class is an auto-generated source file for a HAPI
+ * HL7 v2.x standard structure class.
+ *
+ * For more information, visit: http://hl7api.sourceforge.net/
```

Comment:
We don't need full docs now, but we should update the PR description to include steps on how you generated these (what inputs and tools did you use, etc) so we can review that process as well.

Reply:
So I'm not sure if I would recommend doing that moving forward?  Maybe we can do a quick 16th on this.

I'm also realizing that we should confirm that these actually differ from the base CE